src/Simulation/Simulators/SparseSimulator/SparseSimulatorTests/TestHelpers.cpp

```diff
@@ -0,0 +1,159 @@
+#pragma once
```

Comment:
#pragma once is likely not needed in .cpp files. (Unless they are textually included.)

Reply:
Addressed here: https://github.com/sam-jaques/qsharp-runtime/pull/1